setup/component/setup.py

```diff
@@ -1,6 +1,7 @@
 import setuptools
 
 setuptools.setup(
+    install_requires=['cachetools>=2.0.1'],
     setup_requires=['setuptools-odoo'],
     odoo_addon=True,
```

Comment:
Since that dependency is `__manifest__.py` it's better to do this (otherwise you may end up with duplicates in the generated install_requires.

```python
odoo_addons={
    'external_dependencies_override': {
        'python': {
            'cachetools': 'cachetools>2.0.1',
        }
    }
}
```

Reply:
Makes sense there's a mechanism for that. This what we're looking for? 5f778e0